assets/scss/components/summary-card.scss

```diff
@@ -3,4 +3,9 @@
   @include govuk-media-query($from: "tablet") {
     padding-left: govuk-spacing(2);
   }
+}
+
+.govuk-summary-card__actions {
```

Comment:
is this related to the cypress tests??

Reply:
This is required to conform to the design in Figma.

Reply:
Generally speaking I think it would have been better to have this in a separate PR.